package-lock.json

```diff
@@ -14507,7 +14507,7 @@
       }
```

Comment:
should we place the `package-lock.json` file under `.gitignore` ? It's diff should not be pushed into remotes.

Reply:
Only the root package-lock.json should be updated so this change is correct.

Reply:
hmm, other developers can always do `npm install` inside the repository root. Do we need to push this lock file?

Reply:
This is required because `package-lock.json` installs the exact, versioned dependency tree rather than using starred versioning. Starred versioning can install newer versions of packages that can potentially break the project.